forest/src/cli/config.rs

```diff
@@ -5,18 +5,30 @@ use forest_chain_sync::SyncConfig;
 use forest_libp2p::Libp2pConfig;
 use forest_networks::ChainConfig;
 use serde::{Deserialize, Serialize};
-use std::sync::Arc;
+use std::{collections::HashSet, sync::Arc};
 
 use super::client::Client;
 
-#[derive(Serialize, Deserialize, PartialEq, Default)]
+#[derive(Serialize, Deserialize, Default, Clone, PartialEq)]
+pub struct LogConfig {
+    pub log_values: HashSet<LogValue>,
+}
+
+#[derive(Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
+pub struct LogValue {
+    pub module: String,
+    pub level: String,
+}
+
+#[derive(Serialize, Deserialize, Default, PartialEq)]
 #[serde(default)]
 pub struct Config {
     pub client: Client,
     pub rocks_db: forest_db::rocks_config::RocksDbConfig,
     pub network: Libp2pConfig,
     pub sync: SyncConfig,
     pub chain: Arc<ChainConfig>,
+    pub log_config: Option<LogConfig>,
```

Comment:
What happens if this is not an `Option`?

Reply:
Parse failure

Reply:
Where does this parse failure happen and why does it happen?

Reply:
What would happen if you changed it to:

```suggestion
    pub log_config: LogConfig,
```

Reply:
We will have to write out the internal structure of LogConfig, even when we don't want to add it

Reply:
That sounds like a good thing. The log configuration will still be created from its default value when it's omitted from the configuration file, right?

Reply:
Ok. I'll delete the Option